apollo-api/src/commonMain/kotlin/com/apollographql/apollo3/api/json/MapJsonReader.kt

```diff
@@ -220,11 +241,14 @@ class MapJsonReader(val root: Map<String, Any?>) : JsonReader {
       is String -> JsonNumber(value) // assert value is a valid number
       is JsonNumber -> value
       else -> error("Expected JsonNumber but got $value instead")
+    }.also {
+      consumeValue()
     }
   }
 
   override fun skipValue() {
     nextValue()
+    consumeValue()
```

Comment:
I think you can remove `nextValue` here?

Reply:
Of course! 💪